.github/workflows/tests.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
     strategy:
       matrix:
         arg:
-          - '1.31'
+          - '1.32'
```

Comment:
Don't we want to test against multiple versions for compatibility?

Reply:
We do but Kube 1.32 changes how containers are resized (uses the `resize` subresource) so tests can no longer run against older versions of Kube. Older versions of CSA have already been certified against Kube versions that it supports.